src/main/java/com/team4/testingsystem/controllers/QuestionController.java

```diff
@@ -61,6 +61,12 @@ public QuestionDTO getQuestion(@PathVariable("id") Long id) {
                 .collect(Collectors.toList());
     }
 
+    @ApiOperation(value = "Get content file with questions by it's id")
+    @GetMapping("/listening/{contentFileId}")
+    public ContentFileDTO getListening(@PathVariable("contentFileId") Long contentFileId) {
+        return new ContentFileDTO(contentFilesService.getById(contentFileId));
```

Comment:
I think we need to send listening along with the list of questions

Reply:
Need to discuss it with frontend.
Not sure the player can accept binary file directly